packages/test/test-end-to-end-tests/src/test/localTestSignals.spec.ts

```diff
@@ -189,4 +204,115 @@ describeCompat("TestSignals", "FullCompat", (getTestObjectProvider) => {
 			"client 2 did not receive signal on data store runtime",
 		);
 	});
+
+	// Skipped Tests: Targeted signal functionality is not currently supported by all services
+	describe.skip("Targeted Signals", () => {
+		let user1SignalReceivedCount: number;
+		let user2SignalReceivedCount: number;
+		let user3SignalReceivedCount: number;
+
+		const sendAndVerifyRemoteSignals = async (
```

Comment:
>const sendAndVerifyRemoteSignals = async  [](http://example.com/codeflow?start=2&length=41)

Where possible prefer named functions over lambdas.
I think this could be `async function sendAndVerifyRemoteSignals`.